assets/urls.py

```diff
@@ -23,6 +23,7 @@
 )
 
 urlpatterns = [
+    path('assets/advanced-list/', AssetAdvanceList.as_view(), name="assets-advanced-list"),
```

Comment:
Maybe just have this be the ``assets/`` endpoint?

Reply:
that doesn't work because the POST method (create a new asset) also uses assets/

Reply:
Why does advance-list require POST? It's surely a GET-only request?

Reply:
Ah, I think I see - I mean, why not have GET assets/ point to AssetAdvanceList.list() but keep POST as it is?